docs/docsite/rst/presentations/openstack-modules-intro.rst

```diff
@@ -0,0 +1,394 @@
+==================================
+Intro to Ansible OpenStack Modules
+==================================
+
+This document contains a presentation in `presentty`_ format. If you want to
+walk through it like a presentation, install `presentty` and run:
+
+.. code:: bash
+
+    presentty docs/docsite/rst/presentations/openstack-modules-intro.rst
+
+The content is hopefully helpful even if it's not being narrated, so it's being
+included in the `shade` docs.
+
+.. _presentty: https://pypi.python.org/pypi/presentty
+
+Ansible: Community, Collaboration, and the OpenStack Modules you know and love
+==============================================================================
+
+This talk is Free Software
+==========================
+
+* Written for presentty (https://pypi.python.org/pypi/presentty)
+* docs/docsite/rst/presentations/openstack-modules-intro.rst
+
+Ansible OpenStack Modules
+=========================
+
+* http://docs.ansible.com/ansible/list_of_cloud_modules.html#openstack
+* focused on consuming OpenStack APIs
+* Some are end-user focused
+* Some are deployer focused
+
```

Comment:
May want to keep these bullet points consistent capital/lowercase or:
-Located here:
- All are focused on..
- Some are...etc